Mathlib/Geometry/Group/Growth/LinearLowerBound.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2024 Yaël Dillies, Patrick Luo, Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Patrick Luo, Eric Rodriguez
+-/
+import Mathlib.Algebra.Group.Pointwise.Finset.Basic
+import Mathlib.Algebra.Group.Subgroup.Pointwise
+import Mathlib.Data.Nat.SuccPred
+
+/-!
+# Linear lower bound on the growth of a generating set
+
+This file proves that the growth of a set generating an infinite group is at least linear.
+-/
+
+open Subgroup
+open scoped Pointwise
+
+namespace Finset
+variable {G : Type*} [Group G] [DecidableEq G] {X : Finset G} {n : ℕ}
+
+@[to_additive]
+lemma card_pow_lt_card_pow_succ_of_pow_ne_closure (hX₁ : (1 : G) ∈ X) (hX : X.Nontrivial)
+    (hXclosure : (X ^ n : Set G) ≠ closure (X : Set G)) : #(X ^ n) < #(X ^ (n + 1)) := by
```

Comment:
Actually, can't you strengthen this to a strict subset?

Reply:
Sure yes

Reply:
Seems like that'd be better here and below - up to you if you want to do it before or after this merges

Reply:
Let's do it before